python/instrumentation/openinference-instrumentation-crewai/src/openinference/instrumentation/crewai/_wrappers.py

```diff
@@ -206,6 +212,7 @@ def __call__(
                 raise
             span.set_status(trace_api.StatusCode.OK)
             span.set_attribute(OUTPUT_VALUE, response)
+            span.set_attributes(dict(get_attributes_from_context()))
```

Comment:
We want to set these attributes first, check other implementations if you need more details but you can simply add it to the `start_as_current_span` call